ui/components/app/assets/nfts/nft-details/nft-details.tsx

```diff
@@ -112,7 +111,10 @@ export default function NftDetails({ nft }: { nft: Nft }) {
   const isIpfsURL = nftSrcUrl?.startsWith('ipfs:');
   const isImageHosted =
     image?.startsWith('https:') || image?.startsWith('http:');
-  const nftImageURL = useGetAssetImageUrl(imageOriginal ?? image, ipfsGateway);
+  const nftImageURL = useGetAssetImageUrl(
+    (imageOriginal ?? image) as string | undefined,
+    ipfsGateway,
+  ) as string | undefined;
```

Comment:
Hmm, the return type of this hook is a `string`, is there a need to widen it to `string | undefined`?

Reply:
You're right, this isn't needed. https://github.com/MetaMask/metamask-extension/pull/29445/commits/06687fb379422cf1978fd820f2dd1fd33216cc24